### ISSUE TITLE
Incompatibility with Shopware 5.0.1

### ISSUE THREAD

**Issue by samuelvogel** [VIISON/ShopwareStripePayment#12]
With Shopware 5.0.1 the following error was reported:
```
Fatal error: Declaration of Shopware\Plugins\StripePayment\Components\StripePaymentMethod::validate() must be compatible with ShopwarePlugin\PaymentMethods\Components\BasePaymentMethod::validate(Enlight_Controller_Request_Request $request) in /xyz/engine/Shopware/Plugins/Community/Frontend/StripePayment/Components/StripePaymentMethod.php on line 16
```

**Comment by cjuner** [comment]
Fixed with https://github.com/VIISON/ShopwareStripePayment/commit/bb9846e3f8d0939aa2f8380a087e6ff14858f72e.

Tested in Shopware 5.0.2 and Shopware 5.1.2.

Not released yet.

**Comment by samuelvogel** [comment]
Released with version `1.0.6`.
